graphene_sqlalchemy/tests/test_query.py

```diff
@@ -101,29 +112,23 @@ class Meta:
     class Query(graphene.ObjectType):
         node = Node.Field()
         reporter = graphene.Field(ReporterNode)
-        article = graphene.Field(ArticleNode)
         all_articles = SQLAlchemyConnectionField(ArticleConnection)
 
         def resolve_reporter(self, _info):
             return session.query(Reporter).first()
 
-        def resolve_article(self, _info):
-            return session.query(Article).first()
-
     query = """
-        query ReporterQuery {
+        query {
```

Comment:
Is there a reason to "de-name" the query here ?

Reply:
There was no reason to name the query so I removed to make tests more consistent and simple.